app/models/pglogical_subscription.rb

```diff
@@ -1,36 +1,75 @@
 require 'pg/dsn_parser'
 require 'pg/logical_replication'
+require 'query_relation'
 
-class PglogicalSubscription < ActsAsArModel
-  set_columns_hash(
-    :id                   => :string,
-    :status               => :string,
-    :dbname               => :string,
-    :host                 => :string,
-    :user                 => :string,
-    :password             => :string,
-    :port                 => :integer,
-    :provider_region      => :integer,
-    :provider_region_name => :string
-  )
-
-  def self.find(*args)
```

Comment:
Does this remove the `find` method entirely or is that implemented by one of the included modules?

This is used by the UI

Reply:
Ok, I've added back a `find` alias.